app/src/main/java/org/mozilla/fenix/share/ShareFragment.kt

```diff
@@ -84,13 +117,13 @@ class ShareFragment : DialogFragment(), CoroutineScope {
                     }
                     startActivity(intent)
                 }
-                // TODO support other actions in a follow-up issue
             }
             dismiss()
         }
     }
 
     companion object {
+        // TODO Replace this link with the correct one when provided.
```

Comment:
I'm not really sure what this is supposed to do even. "Add device" was in the mocks, but perhaps we should remove it from the UI for now until it does something useful?

Reply:
Add device was removed from the mocks (in error), so the functionality was missing on what it was supposed to do. This was our backup solution, but some other offline conversations lead us to showing a prompt instead which I'm going to add to this PR.

Reply:
I'll fix this in a follow-up.